terraform/integration/main.tf

```diff
@@ -93,3 +93,10 @@ module "ecr" {
   image_retention_count = 3
 }
 
+module "github_actions_access" {
+  source = "../modules/github_actions_access"
+
+  environment_name = local.environment_name
```

Comment:
does this need a linter running to align on equals?

Reply:
Good spot - forgot to run `fmt` on it (intending to add as a pre-commit hook alongside adding it as a pipeline check, which is the next PR)